docs/Trainer/Checkpointing.md

```diff
@@ -10,13 +11,13 @@ Trainer(default_save_path='/your/path/to/save/checkpoints')
 
 To modify the behavior of checkpointing pass in your own callback.
 
-``` {.python}
+```{.python}
 from pytorch_lightning.callbacks import ModelCheckpoint
 
 # DEFAULTS used by the Trainer
 checkpoint_callback = ModelCheckpoint(
     filepath=os.getcwd(),
-    save_best_only=True,
+    save_top_k=-1,
```

Comment:
so, -1 saves all the checkpoints?
The default should be 1 (or we blow up peoples hard drive)

Reply:
changed to 1